src/Raven.Client/Documents/Subscriptions/SubscriptionWorker.cs

```diff
@@ -603,6 +630,20 @@ private async Task RunSubscriptionAsync()
                         if (ShouldTryToReconnect(ex))
                         {
                             await TimeoutManager.WaitFor(_options.TimeToWaitBeforeConnectionRetry).ConfigureAwait(false);
+
+                            if (_redirectNode == null)
+                            {
+                                var reqEx = _store.GetRequestExecutor(_dbName);
+                                var curTopology = reqEx.TopologyNodes;
+                                var nextNodeIndex = (_forcedTopologyUpdateAttempts++) % curTopology.Count;
```

Comment:
Can this run concurrently?

Reply:
no, the failover does not run concurrently and the _forcedTopologyUpdateAttempts is managed per instance